player/src/main/java/net/pubnative/player/VASTPlayer.java

```diff
@@ -324,7 +325,11 @@ private void setEmptyState() {
         mQuartile = 0;
         mTrackingEventMap = null;
         mProgressTracker = null;
-        mBannerBitmap = null;
+        if (mIsBannerSet) {
```

Comment:
yeah, we need to nullify the bitmap when setting the video, so, you should have a method that uses both things

`setData(VIDEO, BANNER)`

if the banner is nil, not showing It, so there are no reuse conflicts...